File: Unicatólica/Semestre_2024_2/Estrutura_de_Dados/Turma A/Codigos/aula_17092024/exemplo2.cpp
#include <iostream>
using namespace std;

int main(){
    int a = 20;
    int* p = &a;

    cout << "Valor de a: " << a << endl;
    *p = 50;
    cout << "Valor de a: " << a << endl;

    return 0;
}

File: Unicatólica/Semestre_2024_1/POO/Turma A/POO2024A/Aula08_02042024/src/revisao2/Exemplo2.java
package revisao2;

import java.util.Scanner;

public class Exemplo2 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n, soma=0;
		String palavra = "";
		for(int i = 0; i < 5 ; i++) {
			System.out.println("Digite um valor: ");
			n = sc.nextInt();
			soma = soma + n;
		}
		System.out.println("Soma: " + soma);
		
		System.out.println("Digite teste: ");
		palavra = sc.next();
		while(palavra.equals("teste")) {
			System.out.println("OK...");
			System.out.println("Digite teste: ");
			palavra = sc.next();
		}
	}
}
